### DISCUSSION TITLE
Can alm transfer 2nd force constant file to the format of phonopy?

### DISCUSSION THREAD

**Discussion by gangliu-github** [ttadano/alamode#123] [Q&A] [answered]
FC3_SHENGBTE can store 3rd force constant file for ShengBTE. Then, can alm transfer 2nd force constant file to the format of phonopy (FORCE_SETS or FORCE_CONSTANTS)?

**Comment by ttadano** [comment, marked as answer]
While this feature is not supported in alamode, it can be done easily by using the Python wrapper of ALM available at https://github.com/ttadano/ALM. An example of creating a phonopy-readable FC2 file can be found at https://github.com/ttadano/ALM/tree/develop/example/AlN.

**Reply by gangliu-github** [reply]
Thanks. Is ALM another code? I have installed alamode sucessfully. But when I type "import alm" in python, it tells me there is no such module.

**Reply by ttadano** [reply]
Please install the ALM python library following the instruction of https://alm.readthedocs.io/en/develop/.
